docs/proposals/PROPOSAL_C.md

```diff
@@ -0,0 +1,323 @@
+# Proposal C
+
+Create a Node manifest.
+
+## Description
+
+The proposal introduces a "Node" manifest used to organize container images with related artifacts or to group container images together into a higher order artifact. It is based on the typical "Node" data structure used for creating linked lists.
+
+A detailed description of this proposal can be found [here](https://docs.google.com/document/d/1KYJaCm5Z-BJ7hVOI_lDh4vEvrmyEAMwUp2nJBELstGw/edit).
+
+## Links
+
+| Description                                 | Link                        |
+| ------------------------------------------- | --------------------------- |
+| A talk that introduces the idea   	      | [View](https://youtu.be/U_Q5bG-G_Pw) |
+| A Detailed description with pictures        | [View](https://docs.google.com/document/d/1KYJaCm5Z-BJ7hVOI_lDh4vEvrmyEAMwUp2nJBELstGw/edit) |
+
+## Modifications
+
+### JSON Schema
+
+A new manifest called "node" is introduced.
+
+The JSON Schema for a "node" manifest looks like this:
+```jsonc
+{
+  "schemaVersion": 3,
```

Comment:
How do v3 objects relate to v2 objects, especially v2 objects which already exist in the registry?

Are all existing OCI images in a registry also available as v3 Nodes that I can attach to, or do I have to push them as Nodes separately to be able to attach them? This is especially relevant for blobs -- we definitely don't want clients to have to push blobs twice, or for registries to have to store them twice.